.gitignore

```diff
@@ -18,3 +18,4 @@ docs/_build
 *.egg-info
 .vscode
 Pipfile.lock
+venv
```

Comment:
Shouldn't this be .venv?

Reply:
I use the `venv` directory for my virtualenvs. I can change it though.

Reply:
Ah! Apologies, its because I was adding .venv (for https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv/) and thought you were doing similar. No worries.

Reply:
I can move to this convention. Seems to be cleaner to have it in a  "hidden" directory.